src/views/components/ChatMessages.js

```
@@ -147,6 +161,139 @@ export default {
+              content: `<div class="ui fluid">
+                <audio controls style="width: 100%; max-width: 300px;">
+                  <source src="${filePath}" type="audio/mpeg">
+                  <source src="${filePath}" type="audio/ogg">
+                  <source src="${filePath}" type="audio/wav">
+                  Your browser does not support the audio tag.
+                </audio>
+              </div>`
+            };
+
+          case 'document':
+            const sizeText = fileSize ? `(${Math.round(fileSize / 1024)} KB)` : '';
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)